### PR TITLE
Utterance

### LINKED ISSUE #145: Utterance
Add an Utterance class. Support just the following properties for now:

* `transcript` (`Transcription`)
* `transcription` (`Transcription`)
* `translation` (`MultiLangString`)

### PR DESCRIPTION
**Pull request by dwhieb**
## Related Issue

closes #145

## Description

Adds an Utterance class with the following properties:

* `transcript` (`Transcription`)
* `transcription` (`Transcription`)
* `translation` (`MultiLangString`)

## Checklist

- [x] Check out the [Contributing Guidelines][contributing] if you need help getting started with your pull request.

- [x] [Open an issue][issues] for the change (if one doesn't already exist).

- [x] Update tests for planned changes. Check that they are failing. (`npm test`)

- [x] Write code to pass the tests. (`npm test`)

- [x] Add inline code commenting (using [JSDoc][JSDoc] style). Add links to cross-referenced modules, and the DLx data format.

- [x] Generate the developer documentation (`npm run docs`) and check your changes by opening `docs/index.html` in a browser.

[contributing]: https://github.com/digitallinguistics/javascript/blob/master/.github/CONTRIBUTING.md
[issues]:       https://github.com/digitallinguistics/javascript/issues
[JSDoc]:        https://jsdoc.app/

### CODE REVIEW

**Comment by dwhieb** [comment]
[[javascript] Utterance](https://trello.com/c/8A8GurFi/389-javascript-utterance)

File: src/models/index.js [hunk 1]
```diff
@@ -3,10 +3,5 @@
  * @namespace models
```
Comment (vadekh) [inline src/models/index.js:3]: Add Transcription and Utterance references to this file

File: src/models/index.test.js [hunk 2]
```diff
@@ -2,18 +2,16 @@
   max-nested-callbacks,
```
Comment (vadekh) [inline src/models/index.test.js:2]: Add Transcription and Utterance tests to this file as well

**Comment by vadekh** [comment]
@dwhieb I also noticed that this is the only PR where node is upgraded to 14.x was that update to `.github/workflows/test.yml` intentional?

**Comment by dwhieb** [comment]
@vadekh I forgot to update the workflow file to v14 when I updated this library to v14 the other week. As a result some of the tests broke on continuous integration in GitHub Actions, even though they were passing locally. This is because I used some newer syntax ([nullish coalescence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)) that's supported in v14 but not v13.

**Comment by dwhieb** [comment]
@vadekh Addressed all your suggested changes. This one should be good to go! Go and and merge into `master` when you've approved it.

**Pull request merged**